specs/interop/optimism-portal-interop.md

```diff
@@ -0,0 +1,49 @@
+# OptimismPortal
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Overview](#overview)
+  - [Interface and properties](#interface-and-properties)
+  - [Integrating `SharedLockbox`](#integrating-sharedlockbox)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+The `OptimismPortal` contract is upgraded to integrate the `SharedLockbox` and start using the shared ETH liquidity.
+
+### Interface and properties
+
+The `OptimismPortal` contract will add the following storage layout and modified functions:
+
+**`SHARED_LOCKBOX`**
```

Comment:
nit: prefer `####` over `** ... **` so that hyperlinks can be generated to specific sections